### PR TITLE
Updating README Documentation 

### PR DESCRIPTION
**Pull request by axiomcura**
This pull request addresses issue #17 by updating the repository's README. It includes information on our analysis and the datasets utilized.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,87 @@
...
+Next, we applied feature selection using [Pycytominer](https://github.com/cytomining/pycytominer) to obtain informative features.
+This process generated our `feature-selected` profiles, which will be used to train our multi-class logistic regression model.
+
+### Data splitting
+
+We split our data into training, testing, and holdout sets.
+First, we generated our holdout splits, which are reserved exclusively for evaluating the model's performance and are not used during the model's training or tuning.
+
+1. **Plate Holdout Set**: We randomly selected 10 plates.
+2. **Treatment Holdout Set**: We selected one treatment to hold out.
```
Comment (MikeLippincott) [inline README.md]: This might be confusing. Did you hold out only one treatment or was it one treatment per MoA??

Comment (MikeLippincott) [reply README.md]: Or not MoA but injury type in this case

File: README.md [hunk 2]
```diff
@@ -1,10 +1,87 @@
...
-The objective is to identify the number of unique compounds associated with each cell injury and subsequently cross-reference this information with the screening profile. The aim is to assess the feasibility of using the data for training a machine learning model to predict cell injury.
+![workflow](./notebooks/4.visualization/figures/workflow_fig.png)
+> Diagram protraying taken to conduct this study.
 
-## Data Sources
+The objective of this project was to utilize [Pycytominer](https://github.com/cytomining/pycytominer) for generating feature-selected profiles from image-based data, aiming to train a multi-class logistic regression model for predicting cellular injury.
+We obtained the cell-injury dataset from [IDR](https://idr.openmicroscopy.org/webclient/?show=screen-3151) and its corresponding [GitHub repository](https://github.com/IDR/idr0133-dahlin-cellpainting).
+Using Pycytominer, we processed these datasets to prepare them for subsequent model training.
+We trained our model on the cell-injury dataset to predict 15 different types of injuries and our trained model to the JUMP dataset to predict cellular injuries.
+
```
Comment (MikeLippincott) [inline README.md]: I think separating the datasets from the objective would help here.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,87 @@
-# Metadata Search
+# Predicting cellular injury
```
Comment (MikeLippincott) [inline README.md]:
```suggestion
# Predicting cellular injury using Pyctominer
```

**Review by MikeLippincott — APPROVED** [review]
LGTM!

**Comment by axiomcura** [comment]
Thanks! changes have been applied. Now merging!

**Pull request merged**